src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -7,18 +7,26 @@ import ProjectContext from '../../projects/projectContext';
 import ProjectDocsContext from '../../projectInstall/docsContext';
 import ProjectInstallPlatform from '../../projectInstall/platform';
 
+import Raven from 'raven-js';
+
 const Configure = React.createClass({
   mixins: [ApiMixin],
 
+  getInitialState() {
+    return {
+      isFirst: true,
```

Comment:
Also as @MaxBittker suggested, it might be good to clarify variable names, like `isFirstTimePolling` or something and `hasSentRealEvent`